magento/inventory
# MSI-1590: Use Array Key as Sku due to change in array structure

### Description

Previously the SKU was part of the array data, its now been altered so that the SKU is the key for each array of stock data, currently Source Items are not created for newly created products via import.

### Fixed Issues (if relevant)
1. magento-engcom/msi#1590: After Import of products via Catalog Product Import SourceItems are not created in MSI

### Manual testing scenarios
1. Create CSV with new products in Import format
2. Import Products via Admin
3. Check Source Items are successfully created

### Contribution checklist
